packages/schematics/angular/module/files/__name@dasherize@if-flat__/__name@dasherize__.module.ts.template

```diff
@@ -1,13 +1,20 @@
 import { NgModule } from '@angular/core';<% if (commonModule) { %>
-import { CommonModule } from '@angular/common';<% } %><% if (routing) { %>
-
-import { <%= classify(name) %>RoutingModule } from './<%= dasherize(name) %>-routing.module';<% } %>
+import { CommonModule } from '@angular/common';<% } %><% if (lazyRoute && routeDeclarationInlined) { %>
+import { Routes, RouterModule } from '@angular/router';<% } %>
+<% if (routing || lazyRoute && !routeDeclarationInlined) { %>
```

Comment:
I won't extract these two conditions into template variables that are provided from the schematic itself, as they are bloating the template and they are making it hard to comprehend and view the important bit.

`routing || lazyRoute && !routeDeclarationInlined`
`lazyRoute && routeDeclarationInlined`

Reply:
Do you mean that it is better to stick with using the generator `options` and compose the checks inside the template rather than creating new template variables that incorporate them? Sorry, if I didn't understand you correct.

Reply:
I meant template options, example
```ts
      applyTemplates({
        ...strings,
        lazyRoute: routing || lazyRoute && !routeDeclarationInlined,
        ...,
      }),
```

Reply:
I see, but using the `routing` option doesn't necessarily mean that we have a lazy route. Besides, we have both `routing || lazyRoute && !routeDeclarationInlined` && `lazyRoute && routeDeclarationInlined`. 

What about combining just the current `lazyRoute` and `routeDeclarationInlined`? Something like `lazyRouteWithoutRouteMod` and `lazyRouteWithRouteMod`? Still long names though. Do you think that's a good idea?

Reply:
This is good, thought I’d go for the full name, ie: `Module` instead of `Mod` as it’s easier to comprehend.

Reply:
Yeah, it's not worth saving 3 letters in the sake of less comprehensive naming. Will do it :+1: